apis/externalsecrets/v1beta1/secretstore_types.go

```diff
@@ -51,6 +51,9 @@ type ClusterSecretStoreCondition struct {
 
 	// Choose namespaces by name
 	Namespaces []string `json:"namespaces,omitempty"`
+
+	// Choose namespaces by name using glob matching
+	NamespacesGlobs []string `json:"namespacesGlobs,omitempty"`
```

Comment:
Would you add a validation to the admission webhook to make sure the given expression is legitimate?

Reply:
Concerning this one I looked at `pkg/provider/webhook/webhook.go` but not 100% sure to understand how it behave and where I need to plug my check.

Basically it is just a matter of doing:

```golang
if _, err := glob.Compile(pattern) {
  // do something
}
```

Reply:
It should be added [here](https://github.com/external-secrets/external-secrets/blob/54664b43b18ec0571157c1eb5b0b9427a9e9ac67/apis/externalsecrets/v1beta1/secretstore_validator.go#L56-L62) 🙂